src/api/requestsApi.tsx

```diff
@@ -281,13 +293,13 @@ const getPagedRequests = async (
     .items.select(requestedFields)
     .expand(expandedFields)
     .filter(queryString)
-    .top(PAGESIZE)
     .orderBy(
       sortParams.sortColumn?.toString() || "Created",
       sortParams.sortDirection !== "descending"
     )
     .orderBy("Id", true) // Include this or non-unique sort values can cause issues
-    .getPaged();
+    .skip(skiptoken)
```

Comment:
Rather than using skip tokens, we should leverage Async Paging.  Things changed a bit from the previous "getPaged()" function you were using, but very similar.
https://pnp.github.io/pnpjs/concepts/async-paging/